intel_extension_for_transformers/neural_chat/tests/api/test_inference.py

```diff
@@ -46,6 +46,16 @@ def test_retrieval(self):
         response = chatbot.predict("Tell me about Intel Xeon Scalable Processors.")
         print(response)
         self.assertIsNotNone(response)
+    
+    def test_retrieval_accuracy(self):
+        plugins.retrieval.enable = True
+        plugins.retrieval.args["input_path"] = "../../examples/retrieval/Xeon8480Processor.doxc"
```

Comment:
docx

btw, shall we rename to sample.docx?

Reply:
fixed